packages/suite/src/middlewares/onboarding/onboardingMiddleware.ts

```diff
@@ -23,7 +23,12 @@ const onboardingMiddleware =
         }
 
         // initiate on-device tutorial after firmware installation
-        if (action.type === firmwareActions.setStatus.type && action.payload === 'done') {
+        const isOnboardingPage = api.getState().onboarding.reducerEnabled;
```

Comment:
just thinking, what about to use `device.features.initialized` instead of onboarding.reducerEnabled?

Reply:
What's the difference?

Reply:
considering it's in onboardingMiddleware, I also prefer your solution

Reply:
ok 👌